_docs/cardano/2017-01-04-differences.md

```diff
@@ -28,20 +29,23 @@ This document is divided into four parts:
 
 ## Time, Slots, and Synchrony
 
-In a basic model of *Ouroboros* time is divided into discrete units called
+In a basic model of the protocol time is divided into discrete units called
 *slots*. However, there are no details on how to obtain current time securely
 and with enough precision.
 
-In `cardano-sl`, current time is obtained by querying a predefined set of NTP
+In Cardano SL, current time is obtained by querying a predefined set of NTP
 servers. Specifically, each node periodically queries NTP servers and calculates
 mean of results. A node stores last margin (the difference between local time and
 global time) and last obtained global time. А node also stores last slot to ensure
```

Comment:
Oh, this mechanism is implemented, but it's obsolete and is not used currently in production. Instead, we simply rely on users to setup their system time correctly.
We also have a feature to notify users if their time is incorrect (we compare it with NTP time), which is not released yet, but will be released soon.